pkg/e2e/operators/cloudingress/apischeme_cr.go

```diff
@@ -75,6 +81,9 @@ func testCRapischemes(h *helper.H, operatorNamespace string) {
 			err := addApischeme(h, as, operatorNamespace)
 			Expect(err).NotTo(HaveOccurred())
 
+			err = apiSchemeCleanup(h, as.Name)
```

Comment:
Should we just add this as a defer instead?

Reply:
I don't mind if it is.